e2e-tests/utils.js

```diff
@@ -0,0 +1,44 @@
+const { Keyring } = require("@polkadot/api");
+const { Compact } = require("@polkadot/types");
+const { hexToU8a, u8aToHex } = require('@polkadot/util');
+const { cryptoWaitReady } = require('@polkadot/util-crypto');
+class Utils {
+    static TRANSACTION_LENGTH = 145;
+
+    /**
+     * Get Keyring pair prom private key
+     * @param {*} privateKey 
+     */
+    static async getKeyringPair(privateKey){
+        const keyring = new Keyring({ type: 'sr25519' });
+        await cryptoWaitReady();
+        const pair = keyring.addFromSeed(hexToU8a(privateKey));
+        return pair;
+    }
+
+    /**
+     * Sign the transaction with Keyring pair
+     * @param {*} from 
+     * @param {*} to 
+     * @param {*} amount 
+     * @param {*} nonce 
+     */
+    static async signTransaction(from, to, amount, nonce){
+        const len = Compact.encodeU8a(Utils.TRANSACTION_LENGTH);
```

Comment:
Instead of hardcoding the length of the transactions, can't we get it from the `concatStr` variable?